MobileNet - emre570 (#102)
johko/computer-vision-course: chapters/en/Unit 2 - Convolutional Neural Networks/vgg.mdx

# Very Deep Convolutional Networks for Large Scale Image Recognition (2014)

## Introduction

> The VGG architecture was developed in 2014 by Karen Simonyan and Andrew Zisserman from the Visual Geometry Group -and hence named VGG- at Oxford University. The model demonstrated significant improvements over the past models at that time- to be specific 2014 Imagenet challange also known as ILSVRC.

Reviewer: ```suggestion
The VGG architecture was developed in 2014 by Karen Simonyan and Andrew Zisserman from the Visual Geometry Group -and hence named VGG- at Oxford University. The model demonstrated significant improvements over the past models at that time - specifically on the 2014 Imagenet challenge also known as ILSVRC.
```
